apps/web/vibes/2px/components.ts

```diff
@@ -79,6 +79,12 @@ export const components = [
     registryDependencies: [],
     files: ['./components/text-area/index.tsx', '../../lib/utils.ts'],
   },
+  {
+    name: 'popover',
+    dependencies: ['@radix-ui/react-popover', '@/vibes/2px/components/button'],
```

Comment:
`@/vibes/2px/components/button` is a registry dependency

Reply:
Actually it's not used in the component at all.
Maybe you meant to put it in the popover example